pom.xml

```diff
@@ -2700,37 +2706,32 @@
       <dependency>
         <groupId>javax.xml.bind</groupId>
         <artifactId>jaxb-api</artifactId>
-        <version>2.3.1</version>
+        <version>${version.javax.xml.bind}</version>
       </dependency>
       <dependency>
         <groupId>javax.jws</groupId>
         <artifactId>javax.jws-api</artifactId>
-        <version>1.1</version>
+        <version>${javax.jws-api}</version>
```

Comment:
@desmax74 here is missing ${version.javax.jws-api}

Reply:
Fixed